### ISSUE TITLE
Error Loading .mcool files for visualization on Jupyter

### ISSUE THREAD

**Issue by dhrititandon** [higlass/higlass-python#29]
Upon typing in the following command on Jupyter,

from higlass.client import View, Track
from higlass.tilesets import cooler

ts1 = cooler('../data/Dixon2012-J1-NcoI-R1-filtered.100kb.multires.cool')
tr1 = Track('heatmap', tileset=ts1)
view1 = View([tr1])
display, server, viewconf = higlass.display([view1])

display

I get a window which indefinitely says loading and upon changing the panel size I get an error message stating- 
Error retrieving tiles info from : http://localhost:34893/api/v1

**Comment by pkerpedjiev** [comment]
Sorry you're getting that error. I just tried it out in a fresh environment and it worked for me. Just one thing is missing in the docs which is that you need to do `import higlass`.

Here's what I did to set up my environment:

```
conda create --name test-bi python=3.6
source activate test-bi
```

The rest is from the docs:

```
pip install jupyter higlass-python
jupyter nbextension install --py --sys-prefix --symlink higlass
jupyter nbextension enable --py --sys-prefix higlass
jupyter notebook
```

If that doesn't work, may I ask which operating system you are using and how you set up your Python environment?

**Comment by dhrititandon** [comment]
Hi, this is the message I get upon running pip install command

ERROR: higlass-python 0.1.13 has requirement ipywidgets==7.4.2, but you'll
have ipywidgets 7.5.1 which is incompatible

So I think its simply an issue of incompatibility. Thank you for reaching
out though!

**Comment by flekschas** [comment]
This issue might have been resolved in `v0.2.*`. Can you try again and reopen the ticket if the problem persists?
